tsconfig.cjs.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "./tsconfig.base.json",
+  "compilerOptions": {
+    "module": "commonjs",
+    "outDir": "./dist/cjs",
+    "target": "ES5"
+  },
+  "include": [
+    "dist/cjs/src/**/*"
+  ],
+  // XXX: if we don't specify this, tsc adds the value of `outDir` to `exclude`
```

Comment:
> this seems bit odd. I remember it shouldn't be needed with correct config inherits. what brings this for our config?

If I remove this `exclude`, then tsc says: 

> error TS18003: No inputs were found in config file '/Users/tetsuharu/src/js/rxjs/tsconfig.cjs.json'. Specified 'include' paths were '["dist/cjs/src/**/*"]' and 'exclude' paths were '["./dist/cjs"]'.


`npm run copy_src_{cjs,es6}` requires that the source and the dist should be placed in the same file: https://github.com/ReactiveX/rxjs/blob/master/package.json#L75-L76

I feel we might have more refactoring chance to there process. But I also think it should be opened as an another pull request.